### ISSUE TITLE
Docker install option is broken with 2.5.0.0

### ISSUE THREAD

**Issue by clpetersonucf** [ucfopen/Materia-Widget-Dev-Kit#75]
`docker inspect <image>` is now prepending `host_mnt` to the Source path of the mounts. This breaks the host directory path MWDK uses to install widgets.

```
/path/to/materia/fuel/app/tmp/widget_packages
```
is now
```
/host_mnt/path/to/materia/fuel/app/tmp/widget_packages
```

**Comment by FrenjaminBanklin** [comment]
MWDK is also no longer correctly identifying the Docker container that Materia is running in.
Here: https://github.com/ucfopen/Materia-Widget-Dev-Kit/blob/420f5c0d1ca9c75f10aed9fd6c964b700237fda8/express.js#L468

`.*materia-web-base:.*` is no longer a thing - can probably simplify this line to
```javascript
let targetImage = execSync('docker ps -a --format "{{.Image}} {{.Names}}" | grep -e "materia-phpfpm" | head -n 1 | cut -d" " -f2');
```

**Comment by iturgeon** [comment]
I think 'materia-phpfpm' is here to stay ... until we switch to python or something

**Comment by FrenjaminBanklin** [comment]
For what it's worth, widget installation seems to otherwise work just fine between Materia v8.0.0 and MWDK v2.5.0 once I made the change in the above comment.

**Comment by FrenjaminBanklin** [comment]
Looks like this is already fixed in #74.

**Comment by clpetersonucf** [comment]
Fixed in 2.5.1.
